src/test/scala/chiselTests/BulkConnectSpec.scala

```diff
@@ -2,14 +2,14 @@ package chiselTests
 
 import chisel3._
 import chisel3.util.Decoupled
-import chisel3.stage.ChiselStage
+import circt.stage.ChiselStage
 import chisel3.testers.BasicTester
 
 import scala.annotation.nowarn
 
 class BulkConnectSpec extends ChiselPropSpec {
   property("Chisel connects should emit FIRRTL bulk connects when possible") {
-    val chirrtl = ChiselStage.emitChirrtl(new Module {
+    val chirrtl = ChiselStage.emitCHIRRTL(new Module {
```

Comment:
My choice of naming on the `emitCHIRRTL` vs. `emitChirrtl` may have been misplaced given Chisel naming conventions. Out-of-scope of this PR, but we should consider renaming (automatically) `emitCHIRRTL` to `emitChirrtl`.